requirements.txt

```diff
@@ -0,0 +1,3 @@
+Python libraries:
```

Comment:
there's a standard format for this file.  to see what one looks like, type `pip freeze`.  This will ask pip to show you all the libraries and their versions you have installed.  Looking at the imports below, I think the only one you'll need is Flask.  Just put that one line from `pip freeze` in this file and you're good to go.